main.py

```diff
@@ -219,8 +236,11 @@ def parse_input(user_input):
 
 
 @input_error
-def add_contact(args, address_book: AddressBook):
-    name, phone, *_ = args
+def add_contact(address_book: AddressBook):
```

Comment:
Looks good to me. But we need to discuss it.
@ElizPery  Could you please note it to discuss later. I mean behavior for adding contact's data